### PR TITLE
Update our_ambassadors.astro

### PR DESCRIPTION
**Pull request by gpattarone**

### CODE REVIEW

**Comment by mribeirodantas** [comment]
Add a comment here with the photo you want to have on the page. I'll add it for you.

**Comment by gpattarone** [comment]
GiselaPattarone

Thank you!

**Comment by mribeirodantas** [comment]
Could you please check if you're happy with the result? :) https://deploy-preview-332--nextflow-staging.netlify.app/our_ambassadors.html

File: src/pages/our_ambassadors.astro [hunk 1]
```diff
@@ -281,6 +281,18 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
...
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Gisela Pattarone"
+        img="GiselaPattarone.png"
+        country="ar"
+        github="gpattarone"
+        linkedin="giselapattarone"
+      >
+        Gisela is a Bioinformatician at <a href="http://www.cnea.gov.ar/" target="_blank">CNEA</a>, where she designs
```
Comment (mribeirodantas) [inline src/pages/our_ambassadors.astro:291]:
```suggestion
        Gisela is a Bioinformatician at <a href="https://www.argentina.gob.ar/cnea/" target="_blank">CNEA</a>, where she designs
```

**Pull request merged**